src/js/profile/chart.js

```diff
@@ -440,7 +456,7 @@ export class Chart extends Component {
     };
 
     handleChartFilter = (indicators, groups) => {
```

Comment:
same thing here, do we need `previouslySelectedFilters` and `urlParamFilters` separated?